code/modules/admin/topic.dm

```diff
@@ -43,7 +43,7 @@
 					to_chat(usr, "<span class='warning'>Unfortunately there weren't enough candidates available.</span>")
 			if("2")
 				log_admin("[key_name(usr)] has spawned a changeling.")
-				if(!src.makeChanglings())
+				if(!src.makeChangelings())

```

Comment:
`!makeChangelings()`

Reply:
I believe you have to use `src` here because this is called via `Topic()`, but maintainer clarification is probably best.

Reply:
`src.` is never necessary unless you have a conflicting variable (this doesn't). Change this please.